src/components/ui/form/general-info.tsx

```
@@ -0,0 +1,213 @@
+        if (generalInfo.startDate && date && date < generalInfo.startDate) {
+            failToast("Tanggal selesai tidak boleh sebelum tanggal mulai.");
+            return;
+        }
+        setGeneralInfo({
+            ...generalInfo,
+            endDate: date ?? null
+        });
+    };
+
+    // Only update context when input is blurred
+    const handleBlur = (field: keyof GeneralInfo, value: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)